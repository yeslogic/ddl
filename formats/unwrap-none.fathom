def main = {
    let ary : Array64 3 U16 = [1, 2, 3],
    x <- unwrap (array64_find (fun (el : U16) => el == (4 : U16)) ary),
};
