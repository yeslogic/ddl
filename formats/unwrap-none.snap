stdout = '''
def main : Format = {
    let ary : Array16 3 U16 = [1, 2, 3],
    x <- unwrap @U16 (array16_find @3 @U16 (fun (el : U16) => el == (4 :
    U16)) ary),
};
'''
stderr = ''
